.github/workflows/windows.yml

```diff
@@ -74,11 +96,15 @@ jobs:
 
       - name: Tests
         run: |
+          Copy-Item "C:\Program Files (x86)\Intel\oneAPI\mkl\latest\redist\intel64\*.dll" pairinteraction\Release\ -Force -Verbose
```

Comment:
Can you use CMake to copy these DLLs? To my understanding, copying these DLLs is always required under Windows and is nothing specific to our GitHub workflow (MKLConfig.cmake does not take care of these DLLs).

Reply:
The files inside a folder named `redist` better be redistributable.

I'm not a big fan of copying DLLs inside of CMake, because CMake is a buildsystem generator and not a packaging system. Instead `setup.py bdist_wheel` or the `pyinstaller` should copy those DLLs, because these are the programs that actually need them. Didn't we have to copy MKL DLLs in the past when we used Conda on Appveyor? How did we do it back then?

Reply:
> These are all the files in the redist directory

From my point of view, we can copy them all except for the files that correspond to feature that we do not enable (the MKLConfig.cmake provides hints which DLLs correspond to functionality that we do not use like scalapack).